### PR TITLE
vendor: update tipb

### PR DESCRIPTION
**Pull request by coocood**
Add more aggregate functions.

### CODE REVIEW

**Comment by coocood** [comment]
@zz-jason @hanfei1991 @winoros PTAL

File: glide.lock [hunk 1]
```diff
@@ -203,7 +203,7 @@ imports:
   - thrift-gen/zipkincore
   - utils
 - name: github.com/uber/jaeger-lib
-  version: 3b2a9ad2a045881ab7a0f81d465be54c8292ee4f
+  version: c48167d9cae5887393dd5e61efd06a4a48b7fbb3
```
Comment (ngaut) [inline glide.lock]: Do we need to update github.com/uber/jaeger-lib ?

Comment (coocood) [reply glide.lock]: There is no difference in code between the two git version.

Comment (coocood) [reply glide.lock]: I checked again, and there is code difference.

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**